src/handlers/streamHandler.ts

```diff
@@ -292,15 +294,15 @@ export async function handleStreamingMode(
   }
 
   // Convert GEMINI/COHERE json stream to text/event-stream for non-proxy calls
-  if (
-    [
-      //
-      GOOGLE,
-      COHERE,
-      BEDROCK,
-    ].includes(proxyProvider) &&
-    responseTransformer
-  ) {
+  const isGoogleCohereOrBedrock = [GOOGLE, COHERE, BEDROCK].includes(
+    proxyProvider
+  );
+  const isVertexLlama =
+    GOOGLE_VERTEX_AI.includes(proxyProvider) &&
```

Comment:
Sorry for the nitpick: Can we please make this check more strict? Currently, this will allow many other strings which are not vertex-ai. Can cause issues in future.

Reply:
my bad